.expeditor/scripts/shared.sh

```diff
@@ -25,18 +25,31 @@ get_latest_pkg_version_in_channel() {
     echo "${version}"
 }
 
-# Always install the latest hab binary appropriate for your linux platform
+# Install the latest stable `hab` binary using our standard
+# "curl|bash" approach.
 #
-# Accepts a pkg target argument if you need to override it, otherwise
-# will default to the value of `BUILD_PKG_TARGET`
-install_latest_hab_binary() {
+# This is the currently recommended way to get Habitat onto a system
+# that does not already have it.
+#
+# As usual, this will install `hab` at `/bin/hab`
+curlbash_hab() {
     local pkg_target="${1:-$BUILD_PKG_TARGET}"
-    echo "--- Installing latest hab binary for $pkg_target using curl|bash"
+    echo "--- :habicat: Bootstrap installation of the current stable hab binary for $pkg_target using curl|bash"
     # TODO:
     # really weird corner case on linux2 because the 0.82.0 versions of both
     # are the same. let's just delete it
     rm -rf /hab/pkgs/core/hab/0.82.0
     curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash -s -- -t "$pkg_target"
```

Comment:
Could this be simplified by running the script from the repo?    

If we were to break the install script, hopefully running it from the repo would catch it before it makes it to master. If it were to make it to master, then we'd be broken with `curl|bash` until we merged a fix back in.   

The curl adds an additional network hop in the setup of each step, but if curl were to fail it more than likely means GitHub is having issues so avoiding the curl probably wouldn't net us any benefits.

I went down this path in our existing pipeline and ended up not using the install.sh script at all. Unfortunately, I don't appear to have captured this information. From memory, it had to do with the install script not properly updating symlinks to the requested version if there was already a hab on the system.  Something to keep our eye on as we iterate on this.

Reply:
Hrmm... if nothing else, this points out that we should probably have a suite of tests around our `install.sh` script itself.

If you can find the concrete issue's with `install.sh` that you ran into, that would be useful @smacfarlane. At the moment, my inclination is to leave this change "as is", since it's a pretty minimal refactoring, but I'm certainly not opposed to further changes in the future.

Reply:
Yes - let's not conflate these two feature/issues here. That would be a great improvement to log in an issue in the releng repo though.